README.md

```diff
@@ -21,6 +21,12 @@ TRELLO_TOKEN = ##replace with Trello API Token##
 TRELLO_TODO_BOARDID = ##replace with ID of board from Trello to use with this app##
 
 ```
+
+## Travis CI & Docker Hub
+create an account at Travis CI and Docker Hub
+Login to the Travis account and navigate to more options -> Settings
+add an environment variable called DOCKER_USER with a value of your docker hub username
```

Comment:
Is there any particular reason you set this in travis directly rather than in your '.travis.yml` with the other environment variables? Not a major issue, but would be better to be consistent